csharp/ql/consistency-queries/DataFlowConsistency.ql

```diff
@@ -0,0 +1,54 @@
+import csharp
+import cil
+import semmle.code.csharp.dataflow.internal.DataFlowPrivate
+import semmle.code.csharp.dataflow.internal.DataFlowPublic
+import semmle.code.csharp.dataflow.internal.DataFlowImplConsistency::Consistency
+
+private class MyConsistencyConfiguration extends ConsistencyConfiguration {
+  override predicate uniqueEnclosingCallableExclude(Node n) {
+    // TODO: Remove once static initializers are folded into the
+    // static constructors
+    exists(ControlFlow::Node cfn |
+      cfn.getElement() = any(FieldOrProperty f | f.isStatic()).getAChild+() and
+      cfn = n.getControlFlowNode()
+    )
+  }
+
+  override predicate uniqueNodeLocationExclude(Node n) {
+    // Methods with multiple implementations
+    n instanceof ParameterNode
+    or
+    this.missingLocationExclude(n)
+  }
+
+  override predicate missingLocationExclude(Node n) {
+    // Some CIL methods are missing locations
```

Comment:
Is this expected or something that should be fixed?

Reply:
Should probably be fixed.